CHANGELOG.md

```diff
@@ -35,4 +41,5 @@ All notable changes to this service will be documented in this file. The format
   - Security in case of vulnerabilities.
 
 SEE https://keepachangelog.com/en/1.0.0/
--->
\ No newline at end of file
+-->
+>>>>>>> 369a70b1b8723f5911e2c589528a0d8182652bdc
```

Comment:
also here

Reply:
Thanks. I had indeed forgot to save the file after solving the merge conflict. 🙈 
Should be fixed now.